### PR TITLE
Update sklmessages_en_US.properties

### PR DESCRIPTION
**Pull request by its-miroma**
Fix capitalization and wording in English.

### CODE REVIEW

**Comment by its-miroma** [comment]
Hello @indexerrowaty,

This PR for the English file should
be ready for your review.

I have an other ~~unrelated~~ question:
would you accept a PR proposing improvements
for the `README.md` file? A draft of what it
would contain can be found at
<https://github.com/its-miroma/SKlauncher/blob/readme/README.md>.

Thank you

File: assets/launcher/lang/sklmessages_en_US.properties [hunk 1]
```diff
@@ -164,7 +164,7 @@ manager.editor.option.visibility.keep.info=This option is not recommended for lo
 manager.editor.option.jvmpath.name=Java Executable
 manager.editor.option.jvmpath.placeholder=Use bundled Java runtime
 manager.editor.option.jvmargs.name=JVM Arguments
-manager.editor.option.jvmargs.info=This option ignores -Xmx arguments, use Maximum Memory instead
+manager.editor.option.jvmargs.info=This option ignores -Xmx arguments, uses maximum RAM memory instead
```
[the remark this answers is not in the document]
Comment (its-miroma) [reply assets/launcher/lang/sklmessages_en_US.properties]: Sorry for that, I didn't realize that "Maximum RAM Memory" was referring to the option.

**Comment by its-miroma** [comment]
Hello @indexerrowaty,

I have addressed your suggestions, and changed
the wording of `crash.footer.button.link` based
on what I read on #536.

Thank you

File: assets/launcher/lang/sklmessages_en_US.properties [hunk 2]
```diff
@@ -150,9 +150,9 @@ manager.editor.option.resolution.name=Resolution
 manager.editor.option.resolution.height=Height
 manager.editor.option.resolution.width=Width
 manager.editor.option.resolution.fullscreen=Fullscreen
-manager.editor.option.memory.name=Maximum Memory (RAM)
+manager.editor.option.memory.name=Maximum RAM Memory
```
[the remark this answers is not in the document]
Comment (its-miroma) [reply assets/launcher/lang/sklmessages_en_US.properties]: Sorry for that, fixed

**Comment by skmedix** [comment]
I'm not a fan of the capital letters in many of these strings. Many of them were capitalized not so long ago, but it didn't look good

**Comment by its-miroma** [comment]
I appreciate your feedback.

The reason why I proposed the PR is that
many UI guidelines recommend using
capital letters for words in buttons:

- <https://docs.openstack.org/doc-contrib-guide/ux-ui-guidelines/ui-text-guidelines/ui-text-capitalization.html>
- <https://learn.microsoft.com/en-us/style-guide/capitalization#title-style-capitalization>
- <https://www.figma.com/community/plugin/776197163275203567>

I am not a UI designer though, and the
final decision is, of course, yours.

Thank you

**Pull request closed without merging**